### ISSUE TITLE
View button for model should point to Alliance Pathway Viewer

### ISSUE THREAD

**Issue by dustine32** [geneontology/web-gocam#21]
Title explains (most of) it all:
![image](https://user-images.githubusercontent.com/2678599/193695588-ad938341-883a-4a6e-8f44-337b6332bd1d.png)
This View button should go to:
![image](https://user-images.githubusercontent.com/2678599/193695758-8c997172-26fe-4d16-b996-e344707c5dfc.png)
Not to the Noctua editorial interface.

@tmushayahama already has the code change in a branch. I can merge the change and deploy to AWS.

Tagging @thomaspd @kltm

**Comment by kltm** [comment]
@dustine32 @tmushayahama Could you link the branch?

**Comment by kltm** [comment]
Or setup a PR so we can see the code/link changes?

**Comment by dustine32** [comment]
@kltm Ah crud, you're right. Sorry! The branch is creatively named "dev": https://github.com/geneontology/web-gocam/commits/dev

I just realized there were a lot of formatting changes that clutter the changes relevant to this ticket (`noctuaAlliancePathwayViewUrl` env var, `getAlliancePathwayView` function, hook function into `browse-models.component.html`). I also need to remove the [URL change](https://github.com/geneontology/web-gocam/commit/497d288814b6babfa8930da5221b9ace86ac8301) to point back to the regular go-public endpoint for fetching the JSON files. Thinking of just copying the relevant changes to a new branch and PR.

**Comment by kltm** [comment]
No worries, sometimes things get confused. Operationally, the change would be:
```
  noctuaAlliancePathwayViewUrl: 'http://noctua.geneontology.org/workbench/noctua-alliance-pathway-preview/?model_id=',
  noctuaPathwayViewUrl: "http://noctua.geneontology.org/workbench/pathwayview/?model_id=",
```
?

My understanding is that we wanted to have this be an embedded widget in the /go-cam site and not have traffic heading towards the curation system.

**Comment by dustine32** [comment]
Correct on both @kltm. I think (@thomaspd may want to weigh in) this is a short-term fix to at least direct users to a more digestible graph. The full widget embedding could be handled in a new ticket?

**Comment by kltm** [comment]
I think the question here is if this is a "project" or not. If @thomaspd and @pgaudet agree, I'm more than happy to take this as a software maintenance ticket, but we do need to track both of these.

**Comment by thomaspd** [comment]
Yes, this would be fine as a maintenance ticket. I think we're all in agreement that the public GO-CAM site should not point to our curation tool, and it's important to provide a visualization of these GO-CAMs so our users can understand what each of them represents.

**Comment by kltm** [comment]
@thomaspd I think the technical point that I miscommunicated is that this is still pointing to our curation tool and creating system traffic, load, and hinting bots. I agree that this is an improvement (getting rid of the login button if nothing else), but I still think we need to prioritized the "stronger" version of this much sooner than later.

**Comment by kltm** [comment]
@tmushayahama Believes that the stronger version of this--having the widget open on click--is quite easy, so we will take a look at that path first.

**Comment by dustine32** [comment]
@tmushayahama can fill in the details but we discussed (with @thomaspd) that there is some Angular version issue that makes implementing the "on click widget open" much more difficult than expected. So, for a short-term fix, I can PR this change https://github.com/geneontology/web-gocam/commit/f648a8725429ec9ae5fd55da67ddf26a301e3fe1 and deploy.

**Comment by kltm** [comment]
@tmushayahama @dustine32 Yes, it would be good to get the details here for review.

**Comment by kltm** [comment]
Pulling this out of project for keeping traffic away from curation interfaces.
